x-pack/plugins/endpoint/public/plugin.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin } from 'kibana/public';
+import { IEmbeddableStart } from 'src/plugins/embeddable/public';
+import { ResolverEmbeddableFactory } from './embeddables/resolver';
+
+export type EndpointPluginStart = void;
+export type EndpointPluginSetup = void;
+export interface EndpointPluginSetupDependencies {} // eslint-disable-line @typescript-eslint/no-empty-interface
+
+export interface EndpointPluginStartDependencies {
+  embeddable: IEmbeddableStart;
+}
+
+export class EndpointPlugin
+  implements
+    Plugin<
+      EndpointPluginSetup,
+      EndpointPluginStart,
+      EndpointPluginSetupDependencies,
+      EndpointPluginStartDependencies
+    > {
+  public setup() {}
+
+  public start(...args: [unknown, EndpointPluginStartDependencies]) {
+    const [, plugins] = args;
```

Comment:
this seems slightly odd, is there a typescripty way just to do some sort of underscore prefixed argument or something to tell the linter that the argument is unused? That way you don't have to do this sort of odd destructuring?

Reply:
depends entirely on lint rules

Reply:
I don't usually enable this rule. If both args were properties on a single option bag, no rule would complain about us access only 1 of 2 keys. that pattern is rampant in js. therefore I don't see this adding a lot of value. just my own opinion

Reply:
turns out i'm wrong. ts itself now ignores any binding beginning with `_` https://github.com/microsoft/TypeScript/issues/9458